ci-operator/step-registry/dpf/hypervisor/network-tests/dpf-hypervisor-network-tests-commands.sh

```
@@ -100,20 +100,33 @@ else
-# Need to run cmds or script on hypervisor to discover worker node names after being renamed
+
+# Need to export the doca8 dpu-workers after they are renamed, on hypervisor:
+# export TFT_SERVER_NODE=worker-303ea713ea90
+# export TFT_CLIENT_NODE=worker-303ea713ea94
+
+FIND_WORKER_NODES_SCRIPT="/root/doca8/ci/scripts/find_worker_node_names.sh"
+
 if ssh ${SSH_OPTS} root@${REMOTE_HOST} "set -euo pipefail; \
   ls -ltr; \
   env; \
   source ${REMOTE_LAST_OPENSHIFT_DPF_DIR_LOCATION}; \
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)